Dockerfile

```diff
@@ -3,6 +3,10 @@ ARG EXTENSIONS_PATH=/app/phabricator/src/extensions
 COPY differential ${EXTENSIONS_PATH}/differential
 COPY conduit ${EXTENSIONS_PATH}/conduit
 COPY auth ${EXTENSIONS_PATH}/auth
+# Move static resources to phabricator, add files to celerity map array
+COPY auth/auth.css /app/phabricator/webroot/rsrc/css/auth.css
+COPY auth/auth.js /app/phabricator/webroot/rsrc/js/auth.js
+RUN /app/phabricator/bin/celerity map
```

Comment:
This operation requires a database connection which we don't seem to provide in this repo, thus I believe this will need to be executed in another repo; trying to determine what repo that should be now.